services/meta/parameters.md

```diff
@@ -0,0 +1,149 @@
+|------|---------------------------|------------|--------------------------------------------------------------------------------------------------------------|
+| in   | cmd.sup_params.get_report | null       | Requests information about known parameters.                                                                 |
+| out  | evt.sup_params.report     | object     | Reports known parameters of the device, contains array of [`sup_parameter`](#definitions) objects.           |
+| in   | cmd.param.set             | object     | Sets parameter, see [`parameter_value`](#definitions).                                                       |
+| in   | cmd.param.get_report      | str_array  | Requests report with currently set values for specified parameters.                                          |
+| out  | evt.param.report          | object     | Reports current value for specified parameters, contains array of [`parameter_value`](#definitions) objects. |
+
+
+## Service properties
+
+| Name      | Type      | Example | Description                                                |
+|-----------|-----------|---------|------------------------------------------------------------|
+| sup_sizes | int_array | `[1,2]` | Optional. Defines supported sizes of the parameter values. |
+
+## Definitions
+
+* `sup_parameter` is an object of the following structure:
+
+| Field         | Type   | Example                                                                  | Description                                                                    |
+|---------------|--------|--------------------------------------------------------------------------|--------------------------------------------------------------------------------|
+| parameter_id  | string | `"1"`                                                                    | ID of the parameter.                                                           |
+| name          | string | `"Sensitivity of the PR sensor"`                                         | Name of the parameter.                                                         |
+| description   | string | `""`                                                                     | Description of the parameter.                                                  |
+| widget_type   | string | `"select"`                                                               | Format of the parameter, one of [`widget_type`](#definitions).                 |
+| value_type    | string | `"int"`                                                                  | Type of the value.                                                             |
+| options       | object | `[{"label":"Option 1", "value": {"value_type": "int", "int_value": 1}}]` | Only for `select` and `multiselect`. Array of [`select_option`](#definitions). |
```

Comment:
just making sure - this is intended to be english only advanced feature (when implemented in the app)?

Reply:
Yes. Labels will be provided by the producer.